### PR TITLE
OP-536 corresponding GUI changes for core version of OP-536

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-536.

This is the companion PR for the Core PR (https://github.com/informatici/openhospital-core/pull/409); both are required.

### CODE REVIEW

File: src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java [hunk 1]
```diff
@@ -302,8 +302,7 @@ private JPanel getJDataPanel() {
 	 */
 	private JLabel getJCodeLabel() {
 		if (jCodeLabel == null) {
-			jCodeLabel = new JLabel();
-			jCodeLabel.setText(MessageBundle.getMessage("angal.dicomtype.codemaxchars"));
+			jCodeLabel = new JLabel(MessageBundle.getMessage("angal.dicomtype.codemaxchars"));
```
Comment (mwithi) [inline src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java:305]: how many chars? no generalization here?

File: src/main/java/org/isf/disctype/gui/DischargeTypeBrowserEdit.java [hunk 2]
```diff
@@ -302,8 +302,7 @@ private JPanel getJDataPanel() {
 	 */
 	private JLabel getJCodeLabel() {
 		if (jCodeLabel == null) {
-			jCodeLabel = new JLabel();
-			jCodeLabel.setText(MessageBundle.getMessage("angal.common.codemaxchars"));
+			jCodeLabel = new JLabel(MessageBundle.getMessage("angal.disctype.codemaxchars"));
```
Comment (mwithi) [inline src/main/java/org/isf/disctype/gui/DischargeTypeBrowserEdit.java:305]: how many chars? no generalization here?

File: src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java [hunk 3]
```diff
@@ -952,11 +952,11 @@ public void actionPerformed(ActionEvent e) {
 						query.setData(s, true);
 					} catch (IOException err) {
 						JOptionPane.showMessageDialog(null,
-								MessageBundle.getMessage("angal.sql.problemsoccurredwithserverconnection"));
+								MessageBundle.getMessage("angal.sql.problemsoccurredwithserverconnection.msg"));
 						LOGGER.error(err.getMessage(), err);
 					} catch (SQLException err) {
 						JOptionPane.showMessageDialog(null,
-								MessageBundle.getMessage("angal.medicalstock.problemsoccurredwithsqlistruction"));
+								MessageBundle.getMessage("angal.sql.problemsoccurredwiththesqlinstruction.msg"));
```
Comment (mwithi) [inline src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java:959]: this method can be removed I think

File: src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java [hunk 4]
```diff
@@ -300,8 +300,7 @@ private JPanel getJDataPanel() {
 	 */
 	private JLabel getJCodeLabel() {
 		if (jCodeLabel == null) {
-			jCodeLabel = new JLabel();
-			jCodeLabel.setText(MessageBundle.getMessage("angal.preagtreattype.codemaxlength"));
+			jCodeLabel = new JLabel(MessageBundle.getMessage("angal.preagtreattype.codemaxchars"));
```
Comment (mwithi) [inline src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java:303]: how many chars? no generalization here?

File: bundle/language_en.properties [hunk 5]
```diff
@@ -798,7 +994,6 @@ angal.common.description
 angal.common.descriptionm                                                        = DESCRIPTION
```
Comment (mwithi) [inline bundle/language_en.properties:994]: convert to lowercase

File: src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java [hunk 6]
```diff
@@ -327,8 +326,7 @@ private JPanel getJCodeLabelPanel() {
 	 */
 	private JPanel getJDescriptionLabelPanel() {
 		if (jDescriptionLabelPanel == null) {
-			jDescriptionLabel = new JLabel();
-			jDescriptionLabel.setText(MessageBundle.getMessage("angal.preagtreattype.description2"));
+			jDescriptionLabel = new JLabel(MessageBundle.getMessage("angal.preagtreattype.description"));
```
Comment (mwithi) [inline src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java:329]: we can replace with "angal.common.description"?

**Review by mwithi — APPROVED** [review]
Just some comments

File: src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicomtype/gui/DicomTypeEdit.java:305]: Fixed (although GUI changes were not the primary focus of this PR).

File: src/main/java/org/isf/disctype/gui/DischargeTypeBrowserEdit.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/disctype/gui/DischargeTypeBrowserEdit.java:305]: Fixed

File: src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java:303]: Fixed

File: src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java (hunk 6 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/pregtreattype/gui/PregnantTreatmentTypeEdit.java:329]: Replaced with new format:  angal.common.description.txt

File: bundle/language_en.properties (hunk 5 above)
Comment (dbmalkovsky) [reply bundle/language_en.properties:994]: This will be taken care of in my next pass

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Review by mwithi — APPROVED** [review]

**Pull request merged**